new_platforms/docs/GettingStarted.md

```diff
@@ -1,6 +1,72 @@
 Getting Started with Open Enclave: TrustZone and SGX
 =========================================
 
+**Note**: We have migrated the build system to CMake and are in the process of
+updating the documentation accordingly. For the moment, refer to the following
+summary.
+
+## On Windows
```

Comment:
Should we just place this at the top of win_sgx_dev.md? As linux_arm_dev.md looks to be mostly updated.

Reply:
Good point. I'd opt for leaving it on the front page because chances are I missed something on the Linux side. If we move the information under Windows-specific docs, and there are issues with building on Linux, people won't know.